modules/dr-testing-restore-procedures.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/about-disaster-recovery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dr-testing-restore-procedures_{context}"]
+= Testing restore procedures
+
+Testing the restore procedure is important to ensure that your automation and workload handle the new cluster state gracefully. Due to the complex nature of etcd quorum and the etcd Operator attempting to mend automatically, it is often difficult to correctly bring your cluster into a broken enough state that it can be restored. Follow this procedure to test such a broken state.
```

Comment:
```suggestion
Testing the restore procedure is important to ensure that your automation and workload handle the new cluster state gracefully. Due to the complex nature of etcd quorum and the etcd Operator attempting to mend automatically, it is often difficult to correctly bring your cluster into a broken enough state that it can be restored.  To test such a broken state, test your restore procedure.
```

Same comment

> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".

Reply:
I'm just removing this sentence, as changing it feels redundant.